wirecardpaymentgateway/classes/Notification/PostProcessing/Success.php

```diff
@@ -28,6 +29,10 @@ public function process()
         $this->orderAmountCalculator->markParentAsClosedOnFullAmount(
             $this->notification->getParentTransactionId()
         );
-        $this->order_service->createOrderPayment($this->notification->getTransactionId());
+        $this->order_service->createOrderPayment(
+            $this->notification,
+            $this->notification->getRequestedAmount()->getValue(),
+            new OrderAmountCalculatorService($this->order)
```

Comment:
```suggestion
            $this->orderAmountCalculator
```
orderAmountCalculator is already initiated on this phase. No need initiate new instance.